### PR TITLE
Add GET & DELETE /transactions

### PR DESCRIPTION
**Pull request by geokollias**

### CODE REVIEW

**Comment by salamehsameera** [comment]
Thanks George, I think I already added the GET here but it's not merged yet https://github.com/RelationalAI/rai-sdk-python/pull/56 I was waiting on Brad's approval

**Comment by salamehsameera** [comment]
We can completely disregard mine and consider yours, I just added it for tracking the tickets, I can go ahead and close that one without merging

**Comment by geokollias** [comment]
Ah sorry Sameera, totally missed #56. This one adds also the delete, exports the names & adds the permissions.

**Comment by salamehsameera** [comment]
Perfect, thanks. I'll close mine. I just want to point out that it might be better for the reviewers to break down changes in separate PRs instead of having a big one. I would - as a reviewer - prefer to have separate PRs for 1. get 2. delete 3. exporting names 4. adding permissions. It's also better in my opinion to do incremental merges so we can rollback changes that break stuff (if any). But that is just my opinion :)

**Comment by geokollias** [comment]
> Perfect, thanks. I'll close mine. I just want to point out that it might be better for the reviewers to break down changes in separate PRs instead of having a big one. I would - as a reviewer - prefer to have separate PRs for 1. get 2. delete 3. exporting names 4. adding permissions. It's also better in my opinion to do incremental merges so we can rollback changes that break stuff (if any). But that is just my opinion :)

I agree on separate PRs when they make sense. Here the changes are pretty small & trivial, so I thought it's fine. Regarding breaking them, exporting & permissions are tightly coupled to the provided functionality so I'd include them with the actual get or delete functions.

File: examples/delete_transaction.py
Comment (bradlo) [inline examples/delete_transaction.py:1]: Copyright should say 2021-2022 :)

File: railib/api.py
Comment (bradlo) [inline railib/api.py:110]: nit: i try to alphabetize lists like this, easier for developers to find what they are looking for. same for code, unless there is an overriding organizational principle.

**Review by bradlo — APPROVED** [review]
looks great, two minor nitty comments.

File: examples/delete_transaction.py
Comment (geokollias) [reply examples/delete_transaction.py]: Right, done :+1:

File: railib/api.py
Comment (geokollias) [reply railib/api.py]: Good point. Done.

**Pull request merged**
